packages/translations/src/lib/get-dictionary.ts

```diff
@@ -0,0 +1,44 @@
+import 'server-only';
```

Comment:
if the ui-kit is dependent on this, does it mean the components can be rendered only server side?